community-tutorials/install-uptimekuma/01-en.md

```diff
@@ -0,0 +1,136 @@
+author_name: Jonas Löchner
+author_url: https://sancraft.dev
+author_image: https://dl.sancraft.dev/images/Sanoj/1-B.png
+author_bio: -
+tags: [web, mysql, linux, debian, mariadb, uptime-kuma]
+netcup_product_url: https://www.netcup.de/vserver
+language: en
+available_languages: en
+---
+
+# Introduction
+
+In this tutorial you will learn how to install:
+
+- Uptime-Kuma
+
+I am doing this on a Debian 11 server (RS Zimtstern) but the installations should also work on other Debian versions.
+
+# Requirements
+
+You need Apache2, PHP and MariaDB installed on your server see [here](https://github.com/netcup-community/community-tutorials/blob/main/community-tutorials/install-web-server-mariadb-debian-php/01-en.md)
+
+
+## Installing Uptime-Kuma
+
+Uptime Kuma is an open source monitoring tool. [Here is a demo of it.](https://demo.uptime.kuma.pet)<br>
+This tutorial is based on https://github.com/louislam/uptime-kuma/blob/master/README.md <br>
+
+First we need to install Docker:
+
+```sh
+apt remove docker docker-engine docker.io containerd runc docker-compose -y
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/debian $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+apt update && apt install docker-ce docker-ce-cli containerd.io -y
+systemctl enable --now docker
+```
+
+Install Uptime-Kuma:
+
+```sh
+docker volume create uptime-kuma
+docker run -d --restart=always -p 3001:3001 -v uptime-kuma:/app/data --name uptime-kuma louislam/uptime-kuma:1
+```
+
+Now you can open Uptime Kuma via http://IP:3001/ or http://DOMAIN:3001/ (you need to open Port 3001 to the internet!).
+
+## Step 1.1 - Apache setup (optional, but very recommended)
+
+In the following commands, replace every `DOMAIN` with your domain or subdomain.<br/>
+
+- Generate an SSL certificate with `certbot certonly --apache -d DOMAIN`. <br/>
+- Run `nano /etc/apache2/sites-enabled/uk.conf`.<br/>
+
+Now paste these lines into the file:
+
+```
+<VirtualHost *:443>
+    ServerName DOMAIN
+
+    ProxyPass / http://localhost:3001/
+    RewriteEngine on
+    RewriteCond %{HTTP:Upgrade} websocket [NC]
+    RewriteCond %{HTTP:Connection} upgrade [NC]
+    RewriteRule ^/?(.*) "ws://localhost:3001/$1" [P,L]
+
+    ErrorLog ${APACHE_LOG_DIR}/uk.error.log
+
+    SSLCertificateFile /etc/letsencrypt/live/DOMAIN/fullchain.pem
+    SSLCertificateKeyFile /etc/letsencrypt/live/DOMAIN/privkey.pem
+</VirtualHost>
+```
+
+- Replace every `DOMAIN` with your domain and set a folder path under "DocumentRoot". All files in this folder will be available on your DOMAIN in the web.<br/>
+- Save your changes by pressing <kbd>CTRL + X</kbd>, then <kbd>y</kbd> and finally by hitting <kbd>ENTER</kbd>. <br/>
+- Now create this folder with `mkdir /var/www/FOLDER`. <br>
```

Comment:
We don´t need to create this folder, do we?

Reply:
no we need the folder not